.github/workflows/ci.yml

```diff
@@ -74,12 +89,17 @@ jobs:
           echo "::set-env name=CHROME_PATH::$(which google-chrome-stable)"
           echo "::set-env name=POSTGRES_DB_URL::postgres://postgres:postgres@localhost/lighthouse_ci_test"
           echo "::set-env name=MYSQL_DB_URL::mysql://root:mysql@127.0.0.1:33306/lighthouse_ci_test"
+          echo "::set-env name=MYSQL_SOCKET_PATH::/mnt/mysqld.sock"
```

Comment:
fyi @MaximeHeckel you might want to create subdirs so the docker mount doesn't try to clobber an existing directory with the mount I don't think setting `/mnt` for both postgres and mysql will work

Reply:
sorry this is such a pain to debug 😅

Reply:
> you might want to create subdirs so the docker mount doesn't try to clobber an existing directory 

Yeah ran into that issue originally with `/tmp`

Reply:
Is there a way wit gh actions to create the directories before the service starts?

Reply:
Hm I'm not sure to be honest. I would assume there should be but I haven't tried before

Reply:
Did a bit of investigation yesterday night, it doesn't look like it's possible sadly. (Couldn't find a lot of documentations around volumes, tried to work around it but didn't find any satisfying solution to this)
What we can do maybe is use another path than `/mnt`? Do you have any suggestions? (I tried `/tmp` as well but `yarn` wouldn't be able to successfully run after putting the sockets there)

Reply:
have you tried using the workspace dir yet? something like `${{ github.workspace }}/mysql:/var/run/mysqld`? we need it to be predictable and accessible

Reply:
Just gave it a try, but like in the previous attempt the `mysqlsocket` service doesn't seem to be able to start: https://github.com/GoogleChrome/lighthouse-ci/pull/389/checks?check_run_id=919336503